packages/css/src/components/accordion/accordion-section.template.ts

```diff
@@ -0,0 +1,35 @@
+import { Accordion, AccordionSection } from '@dso-toolkit/sources';
+import { html, nothing, TemplateResult } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+
+import { accordionHandleTemplate } from './accordion-handle.template';
+import { accordionTemplate } from './accordion.template';
+
+export function accordionSectionTemplate(accordion: Accordion, section: AccordionSection): TemplateResult {
+  return html`
+    <div class="dso-accordion-section ${classMap({
+      [`dso-${section.state}`]: !!section.state,
+      'dso-open': !!section.open,
+      'dso-nested-accordion': section.subsections?.length ?? 0 > 0
+    })}">
+      ${accordionHandleTemplate(accordion, section)}
+
+      ${section.open
+        ? html`
+          <div class="dso-section-body">
+            ${unsafeHTML(section.children)}
+
+            ${section.subsections
```

Comment:
[Een lege array is nog steeds truthy](https://github.com/dso-toolkit/dso-toolkit/pull/1384#discussion_r771198535)

Ik had 'm moeten #enanderstaart-en... 🥧